Documentation/contributing/development/contributing_guide.rst

```diff
@@ -95,30 +97,44 @@ requirements have been met:
+   between the feature freeze and final release date.
 
-#. Pick the right release-note label
+#. If you have permissions to do it so, pick the right release-note label.
 
-   +--------------------------+---------------------------------------------------------------------------+
-   | Labels                   | When to set                                                               |
-   +==========================+===========================================================================+
-   | ``release-note/bug``     | This is a non-trivial bugfix                                              |
-   +--------------------------+---------------------------------------------------------------------------+
-   | ``release-note/major``   | This is a major feature addition, e.g. Add MongoDB support                |
-   +--------------------------+---------------------------------------------------------------------------+
-   | ``release-note/minor``   | This is a minor feature addition, e.g. Refactor endpoint package          |
-   +--------------------------+---------------------------------------------------------------------------+
+   +--------------------------+--------------------------------------------------------------------------------------------+
+   | Labels                   | When to set                                                                                |
+   +==========================+============================================================================================+
+   | ``release-note/bug``     | This is a non-trivial bugfix and is a user facing bug                                      |
+   +--------------------------+--------------------------------------------------------------------------------------------+
+   | ``release-note/major``   | This is a major feature addition, e.g. Add MongoDB support                                 |
+   +--------------------------+--------------------------------------------------------------------------------------------+
+   | ``release-note/minor``   | This is a minor feature addition, e.g. Add support for a Kubernetes version                |
+   +--------------------------+--------------------------------------------------------------------------------------------+
+   | ``release-note/misc``    | This is a misc change, e.g. Refactor endpoint package, a bug fix of a non-released feature |
+   +--------------------------+--------------------------------------------------------------------------------------------+
 
 #. Verify the release note text. If not explicitly changed, the title of the PR
    will be used for the release notes. If you want to change this, you can add
    a special section to the description of the PR.
+   These release notes are primarily going to be read by users so it is
+   important that release notes for bugs, major and minor features does not
+   contain internal details of Cilium functionality which sometimes are
```

Comment:
```suggestion
   contain internal details of Cilium functionality that are
```
Unless you mean we shouldn't include internal details of Cilium, regardless of whether they are relevant to users?

Reply:
Correct. The release notes should are for users only, for developers there's the whole PR description + commit message.